tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t4.py

```diff
@@ -0,0 +1,159 @@
+"""Schema for Tribal TANF T4 record types."""
+
+from ...util import SchemaManager
```

Comment:
Note: I have moved the SchemaManager in my PR, depending on who merges first we might see a conflict